app/src/main/java/com/wtmberlin/ui/MainActivity.kt

```diff
@@ -3,68 +3,29 @@ package com.wtmberlin.ui
 import android.content.Intent
 import android.net.Uri
 import android.os.Bundle
-import android.view.MenuItem
-import androidx.appcompat.app.ActionBar
 import androidx.appcompat.app.AppCompatActivity
-import androidx.appcompat.widget.Toolbar
-import androidx.core.view.GravityCompat
 import androidx.navigation.findNavController
+import androidx.navigation.ui.AppBarConfiguration
 import androidx.navigation.ui.setupWithNavController
-import com.google.android.material.navigation.NavigationView
 import com.wtmberlin.R
 import kotlinx.android.synthetic.main.main_screen.*
 
 class MainActivity : AppCompatActivity() {
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.main_screen)
-        initToolbar()
-        initNavigationDrawer()
-
-        nav_view.setupWithNavController(findNavController(R.id.nav_host_fragment))
-    }
-
-    private fun initToolbar() {
-        val toolbar: Toolbar = findViewById(R.id.toolbar)
-        setSupportActionBar(toolbar)
-        initActionBar()
-    }
 
-    private fun initActionBar() {
-        val actionbar: ActionBar? = supportActionBar
-        actionbar?.apply {
-            setDisplayHomeAsUpEnabled(true)
-            setHomeAsUpIndicator(R.drawable.ic_menu)
-        }
-    }
+        val navController = findNavController(R.id.nav_host_fragment)
+        val appBarConfiguration = AppBarConfiguration(nav_view.menu, drawer_layout)
 
-    private fun initNavigationDrawer() {
-        val navigationView: NavigationView = findViewById(R.id.nav_view)
-        navigationView.setNavigationItemSelectedListener { menuItem ->
-            menuItem.isChecked = true
-            drawer_layout.closeDrawers()
-            when (menuItem.itemId) {
-                R.id.wtm_website -> displayGlobalCommunityInfo()
-            }
-            true
-        }
+        toolbar.setupWithNavController(navController, appBarConfiguration)
+        nav_view.setupWithNavController(findNavController(R.id.nav_host_fragment))
     }
 
     private fun displayGlobalCommunityInfo() {
         val openURL = Intent(android.content.Intent.ACTION_VIEW)
         openURL.data = Uri.parse("https://www.womentechmakers.com")
```

Comment:
the website is still not launching after the click on the menu

Reply:
Yes, but this problem is not introduced by this change and therefore this change doesn't attempt to fix it. I will submit a fix in a separate PR.